apps/cow-fi/modules/utm/utils.ts

```diff
@@ -6,7 +6,6 @@ const UTM_TERM_PARAM = 'utm_term'
 
 const ALL_UTM_PARAMS = [UTM_SOURCE_PARAM, UTM_MEDIUM_PARAM, UTM_CAMPAIGN_PARAM, UTM_CONTENT_PARAM, UTM_TERM_PARAM]
 
-// import { useLocation, useNavigate } from 'react-router-dom'
```

Comment:
Ignore the changes in this file. It was eslint formatted that auto-fixed all this